### DISCUSSION TITLE
Google Cloudd access key format in config file

### DISCUSSION THREAD

**Discussion by Applevangelist** [DCS-gRPC/rust-server#235] [Q&A] [answered]
What format does 
`-- Your Google Cloud access key.
tts.provider.gcloud.key = "..."` need to be?

**Comment by dogjutsu** [comment, marked as answer]
Looks like:
tts.provider.gcloud.key = "ZNzaQzDNPlzpxYWt_43adOh-Q0crRNTJF1dyCZ8"

Generate in Google cloud under "APIs & Services" -> Credentials ->API Keys (after you create one, hit the 'Show' link).
